applications.yaml

```diff
@@ -173,6 +173,15 @@
   options:
   - border_overflow
   - tray_and_multi_window
+- name: DBeaver
+  identifier:
+    kind: Exe
+    id: dbeaver.exe
+    matching_strategy: Equals
+  float_identifiers:
+  - kind: Class
+    id: '#32770'
```

Comment:
For reference: https://learn.microsoft.com/en-us/windows/win32/winauto/dialog-box

>A dialog box is a temporary window that an application creates to retrieve user input. An application uses dialog boxes to prompt the user for additional information about commands that the user has chosen from a menu. A dialog box contains one or more controls (child windows) with which the user enters text, chooses options, or directs the action of the command.
>
>The window class name for dialog boxes is "#32770".

Perhaps this should be applied as a standalone rule.

Reply:
Thank you for your continued help with my proposed changes. I think I've updated the changes as you've requested. I've removed the application identifier for DBeaver as there is nothing specific to the application left.